static/js/popovers.js

```diff
@@ -853,25 +865,20 @@ exports.register_click_handlers = function () {
         e.preventDefault();
     });
 
-    $("body").on("click", ".set_away_status", (e) => {
-        exports.hide_all();
-        user_status.server_set_away();
-        e.stopPropagation();
-        e.preventDefault();
-    });
-
-    $("body").on("click", ".revoke_away_status", (e) => {
-        exports.hide_all();
-        user_status.server_revoke_away();
+    $("body").on("click", ".status_info", (e) => {
+        const option = $(".status_info option:selected").val();
```

Comment:
instead of `$(".status_info option:selected").val()` you should be able to do `$(this).val()`, so the if condition could probably just be `$(this).val() === "inactive"`. Please don't mark my comments as resolved, if they are not.